jazz_logs/index.js

```diff
@@ -88,14 +88,27 @@ module.exports.handler = (event, context, cb) => {
 
 			querys.push(utils.setQuery("servicename", service));
 			querys.push(utils.setQuery("environment", env));
-			querys.push(utils.setQuery("log_level", logType));
 
 			//Query to filter Control messages
 			querys.push(utils.setQuery("!message", "START*"));
 			querys.push(utils.setQuery("!message", "END*"));
 			querys.push(utils.setQuery("!message", "REPORT*"));
 
-			logger.info("QueryObj: "+ JSON.stringify(querys));
+			if (logType) {
+				var log_type_config = [];
+				for(var count =0; count < config.LOG_LEVELS.length; count++ ) {
```

Comment:
consider using a map here (better for performance in general)
eg:
let log_type_config = config.LOG_LEVELS.map(logLevel => logLevel.Type);
what do you think? you can do this in other place in your code.

Reply:
@svidhani  This actually piqued my interest. For normal array iteration map may be 70%~80% faster compared to native for loop iteration.
https://jsperf.com/sidd-test-for-vs-map

Though, I think the performance with callback almost comes to same I guess.
I have changed in both the places.